### PR TITLE
Update security.he.xlf

### PR DESCRIPTION
**Pull request by ben29**
#38744

### CODE REVIEW

File: src/Symfony/Component/Security/Core/Resources/translations/security.he.xlf [hunk 1]
```diff
@@ -8,51 +8,51 @@
             </trans-unit>
             <trans-unit id="2">
                 <source>Authentication credentials could not be found.</source>
-                <target>Authentication credentials could not be found.</target>
+                <target>פרטי זיהוי לא נמצאו.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Security/Core/Resources/translations/security.he.xlf:11]: Could you double check this? 

Google translated this to "id not found"

File: src/Symfony/Component/Security/Core/Resources/translations/security.he.xlf [hunk 2]
```diff
@@ -8,51 +8,51 @@
...
             </trans-unit>
             <trans-unit id="11">
                 <source>No token could be found.</source>
-                <target>No token could be found.</target>
+                <target>הטוקן לא נמצא.</target>
             </trans-unit>
             <trans-unit id="12">
                 <source>Username could not be found.</source>
-                <target>שם משתמש לא נמצא במערכת</target>
+                <target>שם משתמש לא נמצא במערכת.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Security/Core/Resources/translations/security.he.xlf:47]: FYI, We dont need to write "on system". A simple "Username not found" is good enough. 

Do whatever sounds better.

**Review by Nyholm — COMMENTED** [review]
Awesome. Thank you for this. 

I used google translate to try to confirm this. I could be very wrong in my comments, but can you double check?

File: src/Symfony/Component/Security/Core/Resources/translations/security.he.xlf (hunk 1 above)
Comment (ben29) [reply src/Symfony/Component/Security/Core/Resources/translations/security.he.xlf:11]: In Hebrew it’s hard to translate it
That’s the best

Comment (Nyholm) [reply src/Symfony/Component/Security/Core/Resources/translations/security.he.xlf:11]: Okey. Good. =)

File: src/Symfony/Component/Security/Core/Resources/translations/security.he.xlf (hunk 2 above)
Comment (ben29) [reply src/Symfony/Component/Security/Core/Resources/translations/security.he.xlf:47]: You can remove the word
במערכת

Comment (Nyholm) [reply src/Symfony/Component/Security/Core/Resources/translations/security.he.xlf:47]: Yeah, I think that is better. 

Thank you

**Comment by Nyholm** [comment]
Could you also please add the two missing translations in this file?

```
            <trans-unit id="17">
                <source>Too many failed login attempts, please try again later.</source>
                <target>Too many failed login attempts, please try again later.</target>
            </trans-unit>
            <trans-unit id="18">
                <source>Invalid or expired login link.</source>
                <target>Invalid or expired login link.</target>
            </trans-unit>
```

**Comment by Nyholm** [comment]
Oh, why was this closed?

**Comment by ben29** [comment]
@Nyholm  I did fresh new pull request.

**Pull request closed without merging**